proversity_reports_script/report_backend/completion_report.py

```diff
@@ -1,9 +1,14 @@
-from .base import AbstractBaseReportBackend
+"""
+Report backend for completion report.
+"""
+from collections import OrderedDict
 import csv
+from datetime import datetime
 import os
+
```

Comment:
has this been sorted automatically ? which tool are you using ? this is not easy readable, hadn't you better sort first import, then from..import ?

Reply:
I've only done it manually, but now is addressed, isort was used. @andrey-canon